src/app/services/participants-attendance.service.ts

```diff
@@ -13,10 +14,16 @@ export class ParticipantsAttendanceService {
     //to-do:
     //once the attendance table is merged into master,come back here and configure the right links,
     //data type for the participants/attendance
+
+	getParticipant(eventId: number): Observable<UserDetails[]>{
```

Comment:
We already have AttendanceService. That should be used and this service file deleted